runtime/compiler/control/JITServerCompilationThread.cpp

```diff
@@ -1215,6 +1215,32 @@ TR::CompilationInfoPerThreadRemote::getCachedResolvedMethod(TR_ResolvedMethodKey
    return false;
    }
 
+/**
+ * @brief Method executed by JITServer to retrieve methodinfo from the resolved method cache
+ *
+ * @param key Identifier used to identify a resolved method in resolved methods cache
+ * @param unresolvedInCP The unresolvedInCP boolean value of interest, set by this API
+ * @return returns the method info
+ */
+TR_ResolvedJ9JITServerMethodInfo
+TR::CompilationInfoPerThreadRemote::getCachedMethodInfo(TR_ResolvedMethodKey key, bool *unresolvedInCP)
+   {
+   TR_ResolvedMethodCacheEntry methodCacheEntry = {0};
+
+   if (unresolvedInCP)
+      *unresolvedInCP = true;
```

Comment:
`unresolvedInCP` is always set to true. Is this correct? If so we don't event need to pass `unresolvedInCP` as a parameter.

Reply:
I might have forgotten to look at this. `getCachedMethodInfo` needs to set `*unresolvedInCP = (methodInfoStruct.remoteMirror == NULL)`. Then in `verifyCache` we should trivially return `true` if the client-side method is resolved, but the cached one is not, for the same reasons as given for `isInterpreted`.